lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -122,6 +122,9 @@
     url(r'^gradebook$',
         'instructor.views.api.spoc_gradebook', name='spoc_gradebook'),
 
+    url(r'^gradebook/(?P<page>[0-9]+)$',
+        'instructor.views.api.spoc_gradebook', name='spoc_gradebook'),
```

Comment:
why did you add an extra URL for the same view? Isn't it enough to make the extra part optional?